tests/wat/if.wat

```diff
@@ -0,0 +1,64 @@
+(module
+    (import "wasi_unstable" "fd_write" (func $fd_write (param i32 i32 i32 i32) (result i32)))
+    (import "wasi_unstable" "proc_exit" (func $proc_exit (param i32)))
+    
+    (memory 1)
+    (export "memory" (memory 0))
+
+    (table $tbl 0 anyfunc)
+
+    (data (i32.const 8) "panic\n")
+
+    ;; We also need to manually stub out this function
+    (func (export "__wasm_call_ctors"))
```

Comment:
This is a bug--we should create a soft linked stub in the runtime

Reply:
Created issued #37 to capture this.